.github/actions/setup/action.yml

```diff
@@ -5,16 +5,23 @@ inputs:
   node-version:
     description: 'Node.js version to install'
     default: '18'
+  pnpm-version:
+    description: 'pnpm version to install'
+    default: 8
```

Comment:
I think we should leverage `corepack` and the `packageManager` field in package.json instead of specifying this in the action

The `pnpm/action-setup` action will read this field automatically, so we don't have to specify it here

Reply:
Done